RELEASE_NOTES.md

```diff
@@ -0,0 +1,91 @@
+# Release Notes
+
+## 2.1.1 "Millenium Falcon" - TBD
+
+* This release introduces support for Python 3 (Python 3.5 and higher). It means the agent now
+  supports Python 2.6, 2.7 and >= 3.5.
+
+  On package installation, the installer will try to find a suitable Python version which will
+  be used for running the agent. It defaults to Python 2 (``/usr/bin/env python2``), but if a
+  suitable Python 2 version is not found, it will try to use Python 3 if available (
+  ``/usr/bin/env python3``).
+
+  If you want to select a specific Python version you want to use to run the agent (e.g. you have
+  both Python 2 and Python 3 installed), you can do that using ``scalyr-switch-python`` tool
+  which ships with the agent.
+
+  Selecting Python 2:
+
+  ```bash
+  scalyr-switch-python python2
+  /etc/init.d/scalyr-agent-2 restart
+  ```
+
+  Selecting Python 3:
+
+  ```bash
+  scalyr-switch-python python3
+  /etc/init.d/scalyr-agent-2 restart
+  ```
+
+  Keep in mind that you need to restart the agent after running the ``scalyr-switch-python``
+  command for the changes to take an effect.
+
+* This release adds new ``metric_name_blacklist`` monitor configuration option. With this
+  configuration option you can specify which metrics generated by the monitors should not
+  be logged and shipped to Scalyr.
+
+  For example, if you want to exclude ``app.io.wait`` and ``app.uptime`` metrics for the
```

Comment:
Are these the new metrics you added in?  I think we should call it out specifically in the notes here that two new metrics were added to the `linux_process_metrics` monitor and that if you do not want these metrics, this is what you should do.

Reply:
Yep, agreed, will document that.